pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TransactionEndToEndTest.java

```diff
@@ -365,7 +365,7 @@ public void txnMessageAckTest() throws Exception {
 
         message = consumer.receive(2, TimeUnit.SECONDS);
         Assert.assertNull(message);
-
+        Thread.sleep(500);
```

Comment:
this is not usually a good fix,
you should wait until a given condition is met, other wise the test will be only "less flaky"

Reply:
you are right and i find this problem, add the modify the test to fix this problem